script.js

```diff
@@ -131,6 +131,21 @@ const createListItem = (location, status, lng, lat) => {
   if (location.seekingVolunteers && location.seekingVolunteers.match(/(?:\byes\b)/i)) {
     seekingVolunteers = `<span class="seekingVolunteers location-list--badge">Needs Volunteer Support</span>`
   }
+
+  const openTimeDistribution = moment(location.openingForDistributingDontations, ["h:mm A"])
+  const openTimeDistributionLessOne = moment(location.openingForDistributingDontations, ["h:mm A"]).subtract(1, 'hours')
+  let openingSoonForDistribution = ''
+  if (moment().isBetween(openTimeDistributionLessOne, openTimeDistribution)) {
+    openingSoonForDistribution = `<p class="opening-soon">Opening soon! ${openTimeDistribution.format("LT")} for distributing</p>`
```

Comment:
We'll need to tag these classes for translation -- @caseyhelbling or @andyinabox may have some thoughts on how to do this.

```suggestion
    openingSoonForDistribution = `<p class="opening-soon"><span data-translation-id="opening_soon">Opening soon!</span> ${openTimeDistribution.format("LT")} <span data-translation-id="for_distribution">for distributing</span></p>`
```

Reply:
Updated!